### PR TITLE
Additional notifications for admin/orders.php

### PR DESCRIPTION
**Pull request by lat9**
- "Simplify" the order-listing's navigation, splitting into 3 equal sections for medium/large display.
- Add a notifier to enable additional 'nav' buttons to be inserted to the left or right of the listing navigation.
- Additional notifiers for the order-status-history block, primarily in support of "Ty Package Tracker"

### CODE REVIEW

File: admin/orders.php [hunk 1]
```diff
@@ -747,6 +761,31 @@ function couponpopupWindow(url) {
...
+  // A watching observer can provide an associative array in the form:
+  //
+  // $extra_headings = array(
+  //     array(
+  //       'align' => $alignment,    // One of 'center', 'right', or 'left' (optional)
+  //       'text' => $value
+  //     ),
+  // );
+  //
+  // Observer note:  Be sure to check that the $p2/$extra_headings value is specifically (bool)false before initializing, since
```
Comment (mc12345678) [inline admin/orders.php]: Based on the note of line 828, should this be `$p3/…` instead of `$p2/…` or some other value as these `p` values are not defined anywhere? Also wouldn't below a test of `!empty($extra_headings)` be more encompassing ($extra_headings gets `unset` for example)?

Comment (lat9) [reply admin/orders.php]: Good point; need to remember to review after copy&paste!
